cuequivariance_torch/cuequivariance_torch/primitives/equivariant_tensor_product.py

```diff
@@ -154,6 +155,7 @@ def __init__(
         device: Optional[torch.device] = None,
         math_dtype: Optional[torch.dtype] = None,
         use_fallback: Optional[bool] = None,
+        indexed: Optional[bool] = False,
```

Comment:
@borisfom do you have an idea how we can avoid having a new argument like this?

Reply:
@mariogeiger : Well new argument for the constructor is the least of the issues there. We can’t stay compliant with both compile and script with those kwargs. Indices should be explicit List[Tensor] argument, just like you build it below.